Reinforced.Typings.Cli/Reinforced.Typings.Cli.NETCore.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 	<PropertyGroup>
-		<TargetFramework>net8.0</TargetFramework>
+		<TargetFramework>net9.0;net8.0</TargetFramework>

```

Comment:
@pavel-b-novikov @mark-hingston  Are you sure this works? It is TargetFramework not TargetFramework**s** which would allow for a list of TFMs.

Reply:
Oh. I will anyway fix it by myself before release

Reply:
You're correct it needs to be `TargetFrameworks`

Reply:
Keep in mind that changing from `TargetFramework` to `TargetFrameworks` might impact the output pathes. So it might not be trival change.

Reply:
Actually only one framework is expected at this place. Build script anyway overrides this value